account_analytic_project/README.rst

```diff
@@ -0,0 +1,53 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+========================================
+Project and analytic account integration
+========================================
+
+Gives the option to create projects directly from the analytic accounts they
+should be linked to.
+
+Without this module, if you first create an analytic account and later on you
+decide to have a project as well for this account, the situation is very
+difficult to rectify.
+
```

Comment:
I also don't see the need for the module, as it's very easy to get the same functionality with the standard (although not the best user friendly about the "label" itself). The check creates the project, not the task, so your second point also is meaningless. If you don't want to use task, don't create them under the project.